mslice/app/qapp.py

```diff
@@ -0,0 +1,27 @@
+from mslice.util.qt.QtWidgets import QApplication
+
+qApp = None
+
+
+def create_qapp_if_required():
+    """
+    Create a qapplication for non gui plots
+    """
+    global qApp
+    from mslice.app.presenters import is_gui
```

Comment:
I'm not sure I see the need for the `is_gui` check?

Reply:
I thought that was what we were trying to do here or is that taken care of by the .instance check? I brought it here from plot_figure_manager.